### DISCUSSION TITLE
Can i run the latest kofi version on a ubuntu server vps?

### DISCUSSION THREAD

**Discussion by Grillekkj** [KevinBytesTheDust/CVAmp#244] [Q&A]
need help to do this, if anyone could help

**Comment by jlplenio** [comment]
If I can find a stable solution for all platforms, this will be fixed in the next release.

**Reply by Grillekkj** [reply]
is there a release i can run on a ubuntu server vps?

**Reply by jlplenio** [reply]
You could run the non kofi-version via Python on your linux vps, if you have some coding experience.

**Reply by jlplenio** [reply]
@Grillekkj Linux should work in the newest pre-release. Please try:
https://github.com/jlplenio/crude-twitch-viewer-bot/releases/tag/v0.6.2

**Reply by matt3434** [reply]
Is necessary have ubuntu 22.04 to run v0.6.2? I am getting this error on ubuntu 20.04: Error loading Python lib '/tmp/_MEIX9IRXL/libpython3.11.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `**GLIBC_2.35**' not found (required by /tmp/_MEIX9IRXL/libpython3.11.so.1.0), my GLIBC version is 2.31

**Reply by jlplenio** [reply]
Thank you for providing feedback on the Linux version @matt3434. Hmm, I am not sure how to fix that directly. I think the easiest way is to build an additional executable on ubuntu-20.04, which is offered by GitHub. Should we try this?

**Reply by matt3434** [reply]
At the weekend I'm going to update my Ubuntu version to 22.04 on the VPS and test it again. I'll come back with feedback!

**Reply by matt3434** [reply]
@jlplenio same error for ubuntu 22.04 :\

**Reply by jlplenio** [reply]
Thank you for your feedback. What is your GLIBC version, and can you update it, now that you are on 22.04?
I will build try to build a 20.04 version soon.

**Reply by matt3434** [reply]
As far as I remember the version of Glibc was 2.32. In theory it is possible to update but it is not recommended, I even tried to update but the little time I dedicated to it I was unable to do so.

**Reply by jlplenio** [reply]
Can you try the artifact / version, that you can download from the bottom of this page? This is built on 20.04 - I have not tried it myself. https://github.com/jlplenio/crude-twitch-viewer-bot/actions/runs/8316311398

**Reply by matt3434** [reply]
I was able to run the program without any problems on 20.04
